index.html

```diff
@@ -5,8 +5,231 @@
     <title>HTML Academy: Девайс</title>
   </head>
   <body>
+    <header class="main-header">
+      <nav class="navigation">
+        <form action="" method="get">
+          <input name="s" placeholder="Искать здесь..." type="search">
+          <button type="submit">Найти</button>
+        </form>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Войти</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#" >Сравнить</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Корзина</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#" >Каталог товаров</a>
+          </li>
+        </ul>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#" >Доставка</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Гарантия</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Контакты</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <section class="gallery">
+          <h2 class="gallery-title">Фотогалерея</h2>
```

Comment:
```suggestion
          <h2 class="visually-hidden">Фотогалерея</h2>
```
Заголовок скрыт, значит стилизовать не придется, а значит вместо имени класса просто напишем скрывающий класс